chsdi/lib/files_handler.py

```diff
@@ -121,8 +123,7 @@ class FilesHandler(object):
     def __init__(self, request):
         self.request = request
         # Set up AWS DynamoDB and S3 handlers
-        self.dynamodb_fileshandler = DynamoDBFilesHandler(
-            self.dynamodb_table_name, self.bucket_key_name, self.region)
+        self.dynamodb_fileshandler = DynamoDBFilesHandler(self.dynamodb_table_name, self.bucket_name, self.region)
```

Comment:
Do we use this `bucket_key_name` anywhere?

Reply:
From what I've seen, not anymore.